org.openrgb.OpenRGB.yaml

```diff
@@ -0,0 +1,46 @@
+app-id: org.openrgb.OpenRGB
+runtime:  org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: openrgb
+#rename-icon: springlobby
+#rename-desktop-file: springlobby.desktop
+#rename-appdata-file: springlobby.appdata.xml
+#finish-args:
+#- --share=network
+#- --socket=x11
+#- --share=ipc
+#- --device=dri
+#- --socket=pulseaudio
+
+modules:
+- name: libusb
+  buildsystem: autotools
+  sources:
+  - type: git
+    url: 'https://github.com/libusb/libusb.git'
+    commit: '1a906274a66dd58bf81836db1306902d4a7dc185'
+- name: libhdapi
+  buildsystem: cmake-ninja
+  sources:
+   - type: git
+     url: https://github.com/libusb/hidapi.git
+     commit: 'e9f8c3e75e9891de159aeb18fee1685637c68e84'
+- name: mbedtls
+  buildsystem: simple
+  build-commands:
+   - make install DESTDIR=/app
+  sources:
+  - type: git
+    url: https://github.com/ARMmbed/mbedtls.git
+    commit: 'f71e2878084126737cc39083e1e15afc459bd93d'
+- name: openrgb
+  buildsystem: simple
+  build-commands:
+   - qmake OpenRGB.pro LIBS+=-L/app/lib PREFIX=/app
+   - make install_target install_desktop install_pixmap
+  sources:
+  - type: git
+    url: https://github.com/abma/OpenRGB.git
```

Comment:
why do you use a fork?

Reply:
upstream some changes are needed: https://github.com/CalcProgrammer1/OpenRGB/pull/9

Reply:
usually we add patches

Reply:
atm i expect that the changes are (hopefully) accepted upstream: give it some time please. also i'm new to openrgb, so no clue what i missed.

Reply:
the removal of ` -lstdc++fs` unconditionally will break elsewhere

again the best practice is to use upstream code and then add patches on top as needed.

Reply:
what I mean here is that acceptance of this change as is by upstream is unlikely.

Reply:
the changes where merged upstream, so changed the url to upstream.